### ISSUE TITLE
yandex_mdb_redis_cluster timeout

### ISSUE THREAD

**Issue by warpreality** [yandex-cloud/terraform-provider-yandex#331]
Terraform v1.4.2
on linux_amd64
+ provider registry.terraform.io/yandex-cloud/yandex v0.88.0

I am create a resource yandex_mdb_redis_cluster as same as in documentation, but get error
```
resource "yandex_mdb_redis_cluster" "redis" {
  name             = "redis"
  description      = "Redis service for cache backend"
  environment      = "PRODUCTION"
  tls_enabled      = true
  persistence_mode = "ON"
  network_id       = "${yandex_vpc_network.preproduction_vpc_network.id}"

  config {
    password         = "**********"
    version          = "7.0"
    timeout          = "120"
    databases        = "24"
  }

  resources {
    resource_preset_id = "hm3-c4-m24"
    disk_size          = 50
  }

  host {
    zone      = "ru-central1-a"
    subnet_id = "${yandex_vpc_subnet.preproduction-subnet-a.id}"
  }

  maintenance_window {
    type = "ANYTIME"
  }
}
```

```
Error: error while requesting API to update maintenance window in Redis Cluster "-----": server-request-id = c0d0a908-4148-41fb-8b99-2dad7ebc6278 server-trace-id = 76f04f4bb5abc285:519056242332a5e4:76f04f4bb5abc285:1 client-request-id = 1c2aedd1-8fe2-42a3-8f39-f005e38e3c13 client-trace-id = be6a0117-4aaa-473a-b656-7ffa5ffb3251 rpc error: code = FailedPrecondition desc = no changes detected
```

At the next run terraform request to replace cluster.

**Comment by warpreality** [comment]
found a workaround: if you write the maintenance window in full, it works without errors
```
  maintenance_window {
    type = "WEEKLY"
    day = "SUN"
    hour = 2
  }
```

**Comment by Apelsin234** [comment]
Hello! Thank you for reporting the issue. This problem was fixed in version 0.91.
https://github.com/yandex-cloud/terraform-provider-yandex/blob/master/CHANGELOG.md#0910-may-18-2023
